src/components/inscripcionTurnos/Paso1DDJJ.js

```diff
@@ -13,22 +13,30 @@ import SelectorActividad from '../actividades/SelectorActividad';
 import { actividadesUsuario } from '../../state/usuario';
 import { useRecoilValue } from 'recoil';
 import { DateTime } from 'luxon';
+import { filter } from 'ramda';
 
 export default function Paso1DDJJ({ handleChange, agregarActividad }) {
-  const fechaActual = DateTime.local().toISODate();
-  const actividades = useRecoilValue(
-    actividadesUsuario({ desde: fechaActual })
-  );
+  const fechaActual = DateTime.local().toISOTime();
+  const actividades = useRecoilValue(actividadesUsuario());
```

Comment:
Mi sugerencia es probar esto:

```suggestion
  const actividades = useRecoilValue(actividadesUsuario({ desde: fechaActual }));
```